### PR TITLE
Feat/next automation updated

### PR DESCRIPTION
**Pull request by Fupete**
A long and "weird" merge PR of `next` on `next-automation`

### CODE REVIEW

**Comment by Fupete** [comment]
**Almost ready.** We need to verify a couple of commits, and a couple of bugs, tomorrow during our scheduled meet @astagi @bfabio. Have a good day! 🌞 🥳

File: src/pages/{content.relativePath}.js [hunk 1]
```diff
@@ -745,20 +745,6 @@ export const query = graphql`
             title
           }
           # background
-          buttons {
```
Comment (Fupete) [inline src/pages/{content.relativePath}.js:748]: @bfabio to check

**Comment by bfabio** [comment]
`feat/next-automation` was rebased on `next`

**Comment by Fupete** [comment]
@bfabio do you mean feat/next-automation-updated was rebased on feat/next-automation right?

**Comment by bfabio** [comment]
@Fupete no, `feat/next-automation` was rebased on `next`, so that all commits in feat/next-automation are on top of `next`.

From:
```
                               +---------------------+
                               | feat/next-automation|
+--------------+               |                     |
|     next     |               +---------------------+
+--------------+               |         e4          |
|      h       |               +---------------------+
+--------------+               |         e3          |
|      g       |               +---------------------+
+--------------+               |         e2          |
|      f       |               +---------------------+
+--------------+               |         e1          |
|      e       | ------------> +---------------------+
+--------------+   git branch
|      d       |
+--------------+
|      c       |
+--------------+
|      b       |
+--------------+
|      a       |
+--------------+
```

to 

```
                             +---------------------+
                             | feat/next-automation|
                             |                     |
                             +---------------------+
                             |         h4          |
                             +---------------------+
                             |         h3          |
+--------------+             +---------------------+
|     next     |             |         h2          |
+--------------+             +---------------------+
|      h       |  ---------> |         h1          |
+--------------+             +---------------------+
|      g       |
+--------------+
|      f       |
+--------------+
|      e       |
+--------------+
|      d       |
+--------------+
|      c       |
+--------------+
|      b       |
+--------------+
|      a       |
+--------------+
```

**Pull request closed without merging**